### ISSUE TITLE
Document for BLOB connectivity

### ISSUE THREAD

**Issue by bilencekic** [microsoft/ABAP-SDK-for-Azure#160]
Hello everyone,
I am trying to connect my SAP system to Azure BLOB storage but there is no clear document about this. I did for AWS which has a perfect documentation and working flawlessly but unfortunately i had to make same for ABAP Azure SDK as well for comparison. It has really terrible documentation, screens not matching with existing Azure screens (due to old documents) and for BLOB nothing exists at all. 
Please let me know if there is a clear tutorial and document on this.
Thank you

**Comment by MartinPankraz** [comment]
Hey @bilencekic,

did you check the demo [program package](https://github.com/microsoft/ABAP-SDK-for-Azure/blob/master/src/zadf_demo/zadf_demo_azure_blob.prog.abap)? It has documentation on the header and example code. That was enough for me when I first used it after years of not having programmed ABAP full time.

The SDK is open-source meant for community contributions. It gets better with each developer that puts his mark on it. The philisophy is different to other ABAP SDK software companies. It comes with caveats I agree. Neverthless it only gets better when devs build on each other. Looking forward to constructive feedback and appreciate a PR with improvement :-)

**Comment by bilencekic** [comment]
thanks @MartinPankraz  for the reply. I did exact steps but didint work. First i got SSF errors which i was not able to config any SSF table since it does not come on ZADF_CONFIG. (but on documents it was there as a seperate row to configure SSF keys :D ). i configure based on ABAP code (i was not expecting a documentation there TBH but i tried no luck). I was so close to break my monitor but one more last i tried SAS KEY on top of current URL (after reading 2521 stackoverflow questions and chatgpt prompts), 5 min before my meeting it was working. I was able to post hello world to by container :D .
I will try 1 more time by deleting all configs and redoing again. Else i will create a small azure SDK in my company to push + pull + delete + list documents.

I think there should be a step by step ABAP BLOB document (i am thinking to write article) but with SAS key where we can give range of 50 years of expiry date during SAS key creation.  So signature creation we can pass to Azure instead of re-doing in ABAP. 
Basically we are using SAP RISE on Azure. We had a document storage problem where i suggested AWS but since RISE is on Azure i had to try this. Now i need to test pull/put/delete opreations so can confirm Azure working to proceed using this.
Thank you.

**Comment by MartinPankraz** [comment]
Nice, would amplify the article gladly and even link if feasible :-) cherry on the cake would be a PR on the demo program that would have gotten you where you wanted to be directly.

**Comment by MartinPankraz** [comment]
Hey @bilencekic, did you ever produce the blog? Happy to link it

**Comment by bilencekic** [comment]
hi @MartinPankraz actually i created a nice SDK for azure blob + container services on my local system but i havent write blog. I dont think i will write :D coz many people asking it for free. Sorry it took sometimes for to develop, so i decided not to share hehe. thanks again.

**Comment by MartinPankraz** [comment]
Typical Open source vs. closed source debate. Thanks for the update.
